File: front/Igrac.js
import { Liga } from "./Liga.js";
import { Tim } from "./Tim.js";
export class Igrac {
    constructor(id, ime, prezime, godiste, visina, tim) {
        this.id = id;
        this.ime = ime;
        this.prezime = prezime;
        this.godiste = godiste;
        this.visina = visina;
        this.tim = tim;
    }

    crtaj(host) {

        var tr = document.createElement("tr");
        host.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML = this.ime;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.prezime;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.godiste;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.visina;
        tr.appendChild(el);

    }

}

File: Models/Igrac.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    [Table("Igrac")]
    public class Igrac
    {
        [Key]
        public int ID { get; set; }

        [Required]
        [MaxLength(50)]
        public string Ime { get; set; }

        [Required]
        [MaxLength(50)]
        public string Prezime { get; set; }

        [Required]
        [Range(1980 , 2003)]
        public int Godiste { get; set; }

        public int Visina { get; set; }

        public virtual Tim Tim { get; set; }

        public int TimId { get; set; }

    }
}